src/deployment/bicep-templates/autoscale-settings.bicep

```diff
@@ -5,10 +5,9 @@ param workspaceId string
 param logRetention int
 param autoscale_name string
 param function_diagnostics_settings_name string
-param create_new bool
 
 
-resource autoscaleSettings 'Microsoft.Insights/autoscalesettings@2015-04-01' = if (create_new) {
```

Comment:
Why remove the "if (create_new)" ?

Reply:
Originally they were created twice - once for the python app function and once for net app function. We just create them once now and share them between the two resources. 

This check made sure that this particular call was made only once. That's unnecessary now.